cypress/support/commands.ts

```diff
@@ -1056,7 +1056,7 @@ export const createTaskFromEmpty = (
   interval: string = '24h',
   offset: string = '20m',
   force: boolean = true,
-  delay: number = 10
+  delay: number = 0
```

Comment:
this is also used in the [pinned items test](https://github.com/influxdata/ui/blob/f6e0f246bed706a342a322df0d90e5ae0442c7e1/cypress/e2e/cloud/pinned.test.ts#L144) so we should keep an eye out for any weird behavior there as a result of this change